instrumentation/servlet/servlet-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/ServletRequestParametersExtractor.java

```diff
@@ -77,7 +77,7 @@ private static AttributeKey<List<String>> createKey(String parameterName) {
     // normalize parameter name similarly as is done with header names when header values are
     // captured as span attributes
     parameterName = parameterName.toLowerCase(Locale.ROOT);
-    String key = "servlet.request.parameter." + parameterName.replace('-', '_');
+    String key = "servlet.request.parameter." + parameterName;
```

Comment:
would it be easy to handle new/old semconv here?

Reply:
Done. Not the prettiest, but possible.